node-fs-writefile/writeToFile.js

```diff
@@ -0,0 +1,17 @@
+import { Buffer } from 'node:buffer';
+import { writeFile, open } from 'node:fs/promises';
+import getFileContent from './getFileContent.js';
+
+export default async function writeToFile(file, msg) {
+  try {
+    if (!file) throw new TypeError('writeToFile called with undefined file argument');
+    if (!msg) throw new TypeError('writeToFile called with undefined msg argument');
+    await open(file, 'a') // ensure the file exists so getFileContent doesn't throw an error 🤮
+    let existingText = await getFileContent(file);
+    existingText && (existingText += '\n')
+    const textToWrite = new Uint8Array(Buffer.from(existingText + msg));
+    await writeFile(file, textToWrite);
```

Comment:
If you're going to `open` a file to write to it, it's a good idea to save the handle to a variable so it doesn't immediately close. In the case of appending to an existing file, it'll save you from needing to load the current contents of the file into memory and concatenate them to the new contents.